nixos/modules/services/hardware/throttled.nix

```diff
@@ -27,6 +27,7 @@ in {
       then pkgs.writeText "throttled.conf" cfg.extraConfig
       else "${pkgs.throttled}/etc/throttled.conf";
 
+    hardware.cpu.x86.msr.enable = true;
```

Comment:
Is it better to merge these two config into `hardware.cpu.x86.msr = { ... }`?

Reply:
Stylistic choice. Above this line, systemd is not merged either.

Reply:
> Above this line, systemd is not merged either.

That is different.  Usually, `systemd.packages` and `systemd.services.foo` are not merged.  However, the convention is something like `systemd.services.foo = { ... }` and `hardware.cpu.x86.msr = { ... }`.

> Stylistic choice.

Indeed.